packages/core/content-releases/admin/src/pages/ReleasesPage.tsx

```diff
@@ -192,6 +197,22 @@ const ReleasesPage = () => {
     setReleaseModalShown((prev) => !prev);
   };
 
+  // Check if we have some errors and show a notification to the user to explain the error
+  if (location?.state?.errors) {
+    toggleNotification({
+      type: 'warning',
+      title: formatMessage({
+        id: 'content-releases.pages.Releases.notification.error.title',
+        defaultMessage: 'Your request could not be processed.',
```

Comment:
Did you check with Yannis or Lucas for the message here? It sounds very engineery to me

Reply:
Yeah, they provide me the message